modelci/utils/trt_builder.py

```diff
@@ -0,0 +1,59 @@
+import pycuda.autoinit
```

Comment:
can you explain with more details why is this necessary even you don't use this library?

Reply:
The module pycuda.autoinit, when imported, automatically performs all the steps necessary to get CUDA ready for submission of compute kernels.  My pycharm also can not detect the use of it, but the pycuda cannot perform without this import.

Reply:
We don't need this library.

Reply:
Use TRT runtime engine to do the testing

Reply:
Could you give me more infomation about this?  To perform trt inference with python, in the nvidia official document pycuda is required, while the  blogs i can find also use pycuda to run trt on python.